bin/migrate-files/migrate-files.py

```diff
@@ -33,75 +35,219 @@
+        
+        # Set resume status in case of interuption
+        with open('last-file.pickle', 'wb') as file:
+            pickle.dump(last_document_id, file)
+        
+        cursor.close()
+        conn.close()
+        exit()
+
+    # Display results
+    print("Process complete: Successfully migrated", documents_processed, "files.")
+    return
+
+def planning_docs(starting_planning_document_id,batch,cursor):
+    # Get total number of files
+    try:
+        cursor.execute ('SELECT COUNT(*) FROM OATS.OATS_DOCUMENTS WHERE dbms_lob.getLength(DOCUMENT_BLOB) > 0 AND PLANNING_REVIEW_ID IS NOT NULL')
+    except cx_Oracle.Error as e:
+        error, = e.args
+        print(error.message)
+
+    planning_review_count = cursor.fetchone()[0]
+    print('Count =', planning_review_count)
+
+    # # Execute the SQL query to retrieve the BLOB data and key column
+    cursor.execute(f"""
+    SELECT DOCUMENT_ID, PLANNING_REVIEW_ID, FILE_NAME, DOCUMENT_BLOB, DOCUMENT_CODE, DESCRIPTION, DOCUMENT_SOURCE_CODE, UPLOADED_DATE, WHEN_UPDATED, REVISION_COUNT, dbms_lob.getLength(DOCUMENT_BLOB) as LENGTH
+    FROM OATS.OATS_DOCUMENTS
+    WHERE dbms_lob.getLength(DOCUMENT_BLOB) > 0
+    AND DOCUMENT_ID > {starting_planning_document_id}
+    AND PLANNING_REVIEW_ID IS NOT NULL
+    ORDER BY DOCUMENT_ID ASC
+    """)
+
+    # Track progress
+    documents_processed = 0
+    last_planning_document_id = 0
+
+    try:
+        with tqdm(total=planning_review_count, unit="file", desc="Uploading files to S3") as pbar:
+            while True:
+                # Fetch the next batch of BLOB data
+                data = cursor.fetchmany(batch)
+                if not data:
+                    break
+                # Upload the batch to S3 with a progress bar
+                for document_id, planning_review_id, filename, file, code, description, source, created, updated, revision, length in data:
+                    tqdm.write(f"{planning_review_id}/{document_id}_{filename}")
+
+                    with tqdm(total=length, unit="B", unit_scale=True, desc=filename) as pbar2:
+                        s3.upload_fileobj(file, ecs_bucket, f"migrate/planning_review/{planning_review_id}/{document_id}_{filename}",
+                            Callback=lambda bytes_transferred: pbar2.update(bytes_transferred),)
+                    pbar.update(1)
+                    last_planning_document_id = document_id
+                    documents_processed += 1
+
+    except Exception as e:
+        print("Something went wrong:",e)
+        print("Processed", documents_processed,  "files")
+        
+        # Set resume status in case of interuption
+        with open('last-planning-file.pickle', 'wb') as file:
+            pickle.dump(last_planning_document_id, file)
+        
+        cursor.close()
+        conn.close()
+        exit()
+
+    # Display results
+    print("Process complete: Successfully migrated", documents_processed, "files.")
```

Comment:
Process complete: Successfully migrated planning review. Lets add a little more descriptove comments since there are 3 types of documents that are being migrated here